### PR TITLE
fix USD value not showing

### PR DESCRIPTION
**Pull request by RamRamez**
## Summary by CodeRabbit

- **New Features**
	- Introduced a new action to fill prices for donations without a price in the admin panel.
	- Added functionality to update donation prices and related database entities directly.

- **Refactor**
	- Simplified and streamlined the permissions and actions related to donation price management.

- **Chores**
	- Removed outdated logic and functions related to filling prices for donations in the latest round.

### CODE REVIEW

File: src/server/adminJs/tabs/donationTab.ts [hunk 1]
```diff
@@ -265,6 +269,55 @@ export const buildDonationsQuery = (
   return query;
 };
 
+export const FillPricesForDonationsWithoutPrice = async () => {
```
Comment (mohammadranjbarz) [inline src/server/adminJs/tabs/donationTab.ts:272]: @RamRamez  Can you add a test case for this function?
Create a donation and ,ake the valueUsd of donation as null
Then check if the price has filled or not

**Review by mohammadranjbarz — APPROVED** [review]
Thanks @RamRamez LGTM
But If you add a test case for that would be great

File: src/server/adminJs/tabs/donationTab.ts [hunk 2]
```diff
@@ -265,6 +269,55 @@ export const buildDonationsQuery = (
...
+          message: `${donationsWithoutPrice.length} donations without price found. ${filledPrices.length} prices filled`,
+          type: 'success',
+        },
+      };
+    }
+    donation.valueUsd = donation.amount * price;
+    donation.priceUsd = price;
+    filledPrices.push({ donationId: donation.id, price });
+    await donation.save();
+  }
```
Comment (CarlosQ96) [inline src/server/adminJs/tabs/donationTab.ts:310]: I think we are missing to update the projects totals, but that might be a little heavy. What do you think? @RamRamez

**Review by CarlosQ96 — APPROVED** [review]
LGTM Ramin! As mohammad said, would be good to add tests to the function. And also update the project totals, but that might be heavy on the admin js.

File: src/server/adminJs/tabs/donationTab.ts (hunk 1 above)
Comment (RamRamez) [reply src/server/adminJs/tabs/donationTab.ts:272]: You're right. I added a test case.
Thanks @mohammadranjbarz

File: src/server/adminJs/tabs/donationTab.ts (hunk 2 above)
Comment (RamRamez) [reply src/server/adminJs/tabs/donationTab.ts:310]: Yeah, great point, thanks @CarlosQ96.
I added to update project totals, user total received and user total donated.

**Pull request merged**